HW2_Belyakov/calculator.py

```diff
@@ -0,0 +1,32 @@
+def adding(a, b):
+    return a+b
+
+
+def diff(a, b):
+    return a - b
+
+
+def mult(a, b):
+    return(a*b)
+
+
+def division(a, b):
+    return(a/b)
```

Comment:
здесь где-то скобки есть, где-то нет, с пробелами то же самое
по-хорошему надо единообразно, без скобок и с пробелами вокруг оператора:
```suggestion
    return a / b
```